CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### New features and UX changes
 
 - Add support for non-numerical UID/GID as specified in the appc spec ([#2159](https://github.com/coreos/rkt/pull/2159)).
+- Propagate app exit status to rkt's exit status ([#2198](https://github.com/coreos/rkt/pull/2198).
```

Comment:
Right, I wrote this when I thought we were gonna wait for systemd v229 to be in the coreos flavor. Your write-up sounds good, thanks!